docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -667,6 +713,451 @@ Script.serveFunction("CSK_ModuleName.setSelectedInstance", setSelectedInstance)
 
 Besides, there are also functions to dynamically add/reset the number of instances.
 
+### FlowConfig Functions and Events
+
+If a module should be available within the FlowConfig, there are some events and function it should provide.
```

Comment:
1) either "some events and functions" or "and one specific function"?

2) Is it clear that most /all officially provided modules already support flowconfig and that all the described steps aim at persons who want to enable their selfdeveloped modules in this context? if not I suggest to write the following:

"All modules provided in the GitHub organization already support using them within the FlowConfig. If a module that you develop yourself should be available within the FlowConfig, there are some events and functions it should provide."

Reply:
1) Done
2) For some modules it just does not make sense to be used with FlowConfig (like DateTime, PowerManager, Logger, ...)